client/blocks/reader-full-post/index.jsx

```diff
@@ -319,6 +321,7 @@ export class FullPostView extends React.Component {
 		/*eslint-disable react/jsx-no-target-blank */
 		return (
 			<ReaderMain className={ classNames( classes ) }>
+				{ blogId && <QueryPostLikes siteId={ blogId } postId={ postId } /> }
```

Comment:
I'm not sure if that's possible, but what if only the `feedId` prop is specified and `blogId` isn't? Is it OK then that the post likes are not retrieved? Or can it happen (if at all) only for external RSS feeds that don't have a WP.com site associated with them?

Reply:
In Reader-land `blogId` and `siteId` are regrettably used interchangeably.
If we have a `post` with no `site_ID` then it is for an external rss feed that we don't support liking

Reply:
I think we want to pull these off of the post object, not the post key. blogId will be blank for feed items that have a backing post, no?